src/wagtailtrans/templatetags/languages_wagtail_admin.py

```diff
@@ -0,0 +1,24 @@
+import django
```

Comment:
Can we try not to create too much templatetag files, I think having 2 files one: ``wagtailtrans_tags`` to be used by project implementation the other: ``wagtailtrans_admin_tags`` to store all the tags needed in the wagtail admin?